.github/workflows/release.yml

```diff
@@ -22,7 +22,7 @@ jobs:
         name: Set up Go
         uses: actions/setup-go@v3
```

Comment:
Could you update this to the latest as well just to be on the safe side?

Reply:
Understood, updated [`e6377ec` (#233)](https://github.com/matryer/moq/pull/233/commits/e6377ec6391e9891ab79e85efa6ec4e6497cca90)